manifests/params.pp

```diff
@@ -11,9 +11,12 @@
   $disable_ssl               = false
   $open_ssl_listen_port      = undef
   $postgres_listen_addresses = 'localhost'
-
-  $database                  = 'postgres'
+  
+  $puppetdb_version          = $puppetdb::globals::version
+  $database                  = $puppetdb::globals::database
   $manage_dbserver           = true
+  $manage_pg_repo            = false
```

Comment:
@mullr I think in this case though, it won't work without tweaking for the common RHEL users, that a large part of the user base. That means for PuppetDB, they'll need their own repo managed anyway. Right now the postgresql module can manage these repos to supply a 9.4, I think it's not unrealistic to think that most users will want this. Otherwise, they get a failure, and then have to go off to the docs to turn this on, I can't think of another way they'd want to solve this and have a working PDB stack so it seems like a reasonable default approach. My 2p anyway.